src/RoundTabler/PCIScan.java

```diff
@@ -31,7 +31,7 @@ public PCIScan(){
 		this.pMYSQLColumnTypes = "(mediumtext, longtext, text, tinytext, varchar)";
 	}
 
-	public PCIScan(Connection MYSqlDBConnection, string TableName  ){
+	public PCIScan(Connection MYSqlDBConnection, String TableName  ){
 		this();
 		pDBConnection = DBConnection;
```

Comment:
Seeing errors on my side, I do not think DBConnection exists in Java - maybe just Connection?

Reply:
@theblasher DBConnection is supposed to be a Connections however theres a typo and its pDBConnection. Connection also still has a typo.

Reply:
@zachrussell12 I am getting an error when I pull this into my IDE - are we supposed to be referencing pDBConnection - the variable previously made ?

What is the line supposed to read?

Reply:
yes its meant to referenced DBConnection however there is a typo and it is pDBConnection instead.